index.html

```diff
@@ -43,17 +42,19 @@
             <h1>Chocolate Pizza</h1>
             <h2>POSTED ON 15 DEC 2013 / DESSERTS</h2>
 
-            <p class="right print"><img src="imgs/print.png" alt="print"> PRINT</p>
+            <p class="right print" onclick="printFunction()"><img src="imgs/print.png" alt="print"> PRINT</p>
```

Comment:
Avoid inline JS function calls. It's better practice to assign an id and add an event listener. You wouldn't need to embed an 'img' tag inside a 'p' in this case as well.